Kickstarter-iOS/Views/Controllers/ActivitiesViewController.swift

```diff
@@ -177,7 +177,7 @@ internal final class ActivitiesViewController: UITableViewController {
       }
 
     self.viewModel.outputs.updateUserInEnvironment
-      .observeValues { [weak self] user in
+      .observeValues { user in
```

Comment:
Weird, this should have been fixed on `master` 🤔

Reply:
Hmm.. so maybe the feature-native-checkout branch is outdated?